src/systems/ec/specs/components/gallery/demo/data.js

```diff
@@ -23,10 +23,6 @@ module.exports = {
       alt: 'Image 3',
       description: 'Living, working, travelling in the EU',
       meta: 'Copyright, Author, Licence for image 3',
-      icon: {
-        shape: 'general--audio',
```

Comment:
whhhaaaat? you haven't added support for audio files??!! 😱

Reply:
not in the specs :)